src/test/UniswapV3SplitRouteCallee.t.sol

```diff
@@ -0,0 +1,375 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+// Copyright (C) 2023 Maker Ecosystem Growth Holdings, INC.
+// Copyright (C) 2023 Dai Foundation
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published
+// by the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+pragma solidity =0.6.12;
+pragma experimental ABIEncoderV2;
+
+import "ds-test/test.sol";
+import "dss-interfaces/Interfaces.sol";
+import {UniswapV3SplitCallee} from "src/UniswapV3SplitRouteCallee.sol";
+
+import "dss/clip.sol";
+import "dss/abaci.sol";
+
+interface Hevm {
+    function warp(uint256) external;
+    function rollFork(uint256) external;
+    function store(address c, bytes32 loc, bytes32 val) external;
+    function load(address, bytes32) external returns (bytes32);
+    function envOr(string calldata, uint256) external returns (uint256);
+    function envOr(string calldata, address) external returns (address);
+    function envOr(string calldata, bytes calldata) external returns (bytes memory);
+}
+
+struct UniswapV3ExactInputParams {
+    bytes path;
+    address recipient;
+    uint256 amountIn;
+    uint256 amountOutMinimum;
+}
+
+interface WethAbstract is GemAbstract {
```

Comment:
All weth related stuff are unused, please remove as in the 1inch test.

Reply:
Weth stuff removed.